mscore/palettebox.cpp

```diff
@@ -191,18 +191,23 @@ void PaletteBox::paletteCmd(PaletteCommand cmd, int slot)
       switch(cmd) {
             case PaletteCommand::PDELETE:
                   {
-                  vbox->removeItem(item);
-                  b->deleteLater();      // this is the button widget
-                  delete item;
-                  item = vbox->itemAt(slot);
-                  vbox->removeItem(item);
-                  delete item->widget();
-                  delete item;
-                  for (int i = 0; i < (vbox->count() - 1) / 2; ++i)
-                        static_cast<PaletteBoxButton*>(vbox->itemAt(i * 2)->widget())->setId(i*2);
-                  emit changed();
+                  QMessageBox::StandardButton reply;
+                  reply = QMessageBox::question(this," Warning "," Are you sure ? ",QMessageBox::Yes | QMessageBox::No);
```

Comment:
use `... QWidget::tr("Warning"), QWidget::tr"Are you sure?"` ... to (a) make the textx translatable, (b) remove the superfluos spaces